### PR TITLE
@benisgold/ens i18n

### PR DESCRIPTION
**Pull request by benisgold**
Fixes RNBW-2621

## What changed (plus any additional context for devs)
i18n
## PoW (screenshots / screen recordings)
n/a
## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels?

### CODE REVIEW

File: src/components/ens-registration/SearchResult/SearchResultGradientIndicator.tsx [hunk 1]
```diff
@@ -28,19 +29,19 @@ const SearchResultGradientIndicator = ({
...
         gradient = colors.gradients.transparentToLightOrange;
       } else {
-        text = '🥳 Available';
+        text = `🥳 ${lang.t('profiles.search.available')}`;
         gradient = colors.gradients.transparentToGreen;
       }
       break;
     case 'expiration':
-      text = `Til ${expirationDate}`;
+      text = `${lang.t('profiles.search.until')} ${expirationDate}`;
```
Comment (jxom) [inline src/components/ens-registration/SearchResult/SearchResultGradientIndicator.tsx:40]: idk, but should we do interpolation here? there could be other languages that could have "til" on the right hand side... i guess it is good practice to interpolate string variables in i18n generally

```suggestion
      text = `${lang.t('profiles.search.until', { content: expirationDate })}`;
```

File: src/languages/_english.json [hunk 2]
```diff
@@ -544,6 +544,23 @@
...
     },
+    "profiles": {
+      "search": {
+        "header": "Find your name",
+        "description": "Search available profile names",
+        "available": "Available",
+        "taken": "Taken",
+        "registered_on": "This name was last registered on",
+        "year": "Year",
+        "until": "Til",
```
Comment (jxom) [inline src/languages/_english.json:555]:
```suggestion
        "until": "Til %{content}",
```

File: src/screens/ENSSearchSheet.js [hunk 3]
```diff
@@ -155,22 +156,23 @@ export default function ENSSearchSheet() {
               <Inset horizontal="19px">
                 {isRegistered ? (
                   <Text color="secondary50" size="16px" weight="bold">
-                    This name was last registered on{' '}
-                    {registrationData?.registrationDate}
+                    {`${lang.t('profiles.search.registered_on')} ${
```
Comment (jxom) [inline src/screens/ENSSearchSheet.js:159]: same here?

**Review by jxom — APPROVED** [review]

**Review by estebanmino — APPROVED** [review]
🚀

**Pull request merged**